.circleci/config.yml

```diff
@@ -78,6 +78,9 @@ jobs:
           command: |
             ./standalone/swiftgen --version
             ./standalone/swiftgen templates list
+      - run:
```

Comment:
I'd actually prefer if this was a separate "job"  and we add it to the workflow. See the StencilSwiftKit config:
https://github.com/SwiftGen/StencilSwiftKit/blob/master/.circleci/config.yml